x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/nio/SSLOutboundBuffer.java

```diff
@@ -63,8 +60,13 @@ FlushOperation buildNetworkFlushOperation(BiConsumer<Void, Exception> listener)
         }
 
         return new FlushOperation(byteBuffers, (r, e) -> {
-            IOUtils.closeWhileHandlingException(pagesToClose);
-            listener.accept(r, e);
+            try {
+                IOUtils.close(pagesToClose);
+            } catch (Exception ex) {
+                ex.addSuppressed(e);
```

Comment:
`e` is normally `null` which I think is forbidden to suppress?

Reply:
wops right, fixed :)